### DISCUSSION TITLE
Improve compatibility with Sliding Panes

### DISCUSSION THREAD

**Discussion by pikacho13** [davish/obsidian-full-calendar#392] [Feature Requests]
Thanks for making this awesome plugin! I haven't started using it yet but I'm really excited to give it a go!

I've just come across a problem of not being able to view it in full screen. When I click the plugin on the side, it opens it as an additional window and I can't seem to close the original pane that was open.

I've tried to search but can't find anything about this online (to be honest I'm not too sure what's the best way to search this issue so that's likely the problem).

If someone could help me figure out how to open this for full view, that'd be much appreciated!
![screenshot 1](https://user-images.githubusercontent.com/65567044/182424031-ed981d14-e61c-4014-bdc6-f15d4e63c641.png)

**Comment by pikacho13** [comment]
Also, when I open full calendar, it will close other notes that are open, and basically look like the above screen. and while full calendar's open, any notes I open will open up on the next pane even when I don't choose to open it in a new pane

**Comment by cdmoss** [comment]
Also experiencing this. I hope these little things get cleaned up, because the rest of this plugin is brilliant and would be the last puzzle piece for fixing my long suffering organisation system!

**Comment by MatteoGalletta** [comment]
I was able to replicate the issue by installing the plugin **Sliding Panes (Andy's Mode)**, uninstalling (or just disabling) it should work, at least until the incompatibility gets fixed.

**Comment by cja-github** [comment]
I have this problem with Full Calendar and Sliding Panes (Andy's Mode) too

**Comment by davish** [comment]
I personally don't use Sliding Panes, and would appreciate any help in debugging this!

**Comment by Arecsu** [comment]
I experience a similar problem. When I move the calendar to a new window (a child to the main window), I can't create new tasks. It freezes as soon as I click. Can't drag the click to create a time range. In the main Obsidian window though, the create new event dialog spawns.

Maybe this is a related issue

**Comment by FelixBrakel** [comment]
I have the same issue as @Arecsu. When I drag the calendar tab outside the Obsidian window to create a new window there are some issues when creating events. I can click to start creating a new one but the dialog window only shows up when I focus on the main Obsidian window again. This also has the side effect of dragging the event around as if it is still selected.
